app/src/services/subscribers/CensusEventSubscriberService.ts

```diff
@@ -119,18 +118,17 @@ export default class CensusEventSubscriberService implements ServiceInterface {
         });
     }
 
-    private checkAlert(event: PS2ZoneEvents): boolean {
+    private alertExists(worldId: string, zoneId: string): boolean {
         return this.activeAlerts.alertExists(
-            parseInt(event.world_id, 10),
-            parseInt(event.zone_id, 10),
+            parseInt(worldId, 10),
+            parseInt(zoneId, 10),
         );
     }
 
-    // Checks for compatible event types to see if alert is running on that World + Zone combo
-    private getAlert(event: PS2ZoneEvents): ActiveAlertInterface {
+    private getAlert(worldId: string, zoneId: string): ActiveAlertInterface {
```

Comment:
Ditto with this one, send parsedInts to the `getAlert` method, as I think this is the only place it's called.